### DISCUSSION TITLE
Failed to load resource: the server responded with a status of 404 (Not Found) livewire.js not found

### DISCUSSION THREAD

**Discussion by Gopal-Bandyopadhyay** [livewire/livewire#5949] [1. Help] [answered]
most of the developer getting error when installing livewire version 3 and try to run counter example and following error received.

Failed to load resource: the server responded with a status of 404 (Not Found) livewire.js

**Comment by joshhanley** [comment]
@Gopal-Bandyopadhyay thanks for reporting! Can you provide details about what operating system you are on, what web server you are using, and what your web server configuration is? Thanks!

**Comment by Gopal-Bandyopadhyay** [comment]
windows 10 and wamp server

**Comment by Gopal-Bandyopadhyay** [comment]
Screenshot is attached for your reference.
After several google search I have go through this instruction also
https://studentsdisha.com/how-to-install-livewire-3/

**Comment by Gopal-Bandyopadhyay** [comment]
may see this attachment.

**Comment by joshhanley** [comment]
@Gopal-Bandyopadhyay I don't see your screenshot. Also I just noticed that you have opened a duplicate issue #5929 19 hours ago. Please don't open multiple issues in the future.

**Comment by Gopal-Bandyopadhyay** [comment, marked as answer]
Problem resolved after putting following code.

Livewire::setUpdateRoute(function ($handle) {
   return Route::post('Autogen_V3_Livewire_3/public/livewire/update',
$handle);
});

Livewire::setScriptRoute(function ($handle) {
   return Route::get('Autogen_V3_Livewire_3/public/livewire/livewire.js',
$handle);
});
